### ISSUE TITLE
Hotkey to take a break

### ISSUE THREAD

**Issue by Anon4Lulz** [rcaelers/workrave#262]
It'd be a cool to have some hotkeys which whom you can operate

**Comment by pentzin** [comment]
Agreed, I was looking for a global hotkey to quickly hide the rest break warning dialog (during presentations over Teams meetings)

**Comment by michaelmhoffman** [comment]
There used to be some keyboard control available with the existing interface on Windows, using something like Win+B to go to the notification area (aka "system tray"), selecting the Workrave icon with arrow keys, Shift+F10 to open the menu, and then selecting a menu item. In Workrave 1.10.50 on Windows 11, I don't have any keyboard control over this context menu, but issues make it look like this might have been fixed for Workrave 1.11?
